server/controllers/setup.js

```diff
@@ -53,6 +55,129 @@ router.get('/', function(req, res) {
     daily.save();
     evening.save();
 
+    var question1 = new Question({
```

Comment:
Because file setup.js provides to filling mongoDB by all needed data
Then from another file from folder CONTROLLER through some route we use data from mongoDB